build/manage-image-lifecycle.ps1

```diff
@@ -0,0 +1,45 @@
+$buildNumber = "$(Release.Artifacts.CIBuild.BuildNumber)"
```

Comment:
Should this be a powershell function or module? Thoughts on how we call this script?

Reply:
This script (or inline script) should be called a new stage immediately after the "Docker Release" stage in the "[Publish Release](https://microsofthealthoss.visualstudio.com/FhirServer/_release?definitionId=19&view=mine&_a=releases)". That line is from the "Docker Release" stage where it pulls the source image from the registry. The new stage will install Oras and execute the script to attach the lifecyle/lineage metadata to images in the repositories.

I am trying to find a way to test the above by creating my test release + my test container registry. I will let you know how it goes.